### PR TITLE
Update page-officers.hbs -- YASH

### PR DESCRIPTION
**Pull request by yash-dubs**

### CODE REVIEW

**Review by encadyma — CHANGES_REQUESTED** [review]
Thanks for the PR!

Although I see your changes to page-officers.hbs, you will also need to include your officer photo as well. Be sure to `git add` the picture at the proper location and make another commit, then push again to your branch.

File: page-officers.hbs [hunk 1]
```diff
@@ -314,6 +314,15 @@ into the {body} of the default.hbs template --}}
                     <a target="_blank" class="icon-github" href="https://github.com/johnlee-jh"></a>
                   </div>
                 </div>
+		<div class="ieee-officer-card">
```
Comment (encadyma) [inline page-officers.hbs]: nit: fix indentation so that it's consistent with previous lines

**Review by PotatoParser — APPROVED** [review]
Looks great!

**Review by encadyma — APPROVED** [review]
Looks good!

**Pull request merged**